fedimint-core/src/db/mod.rs

```diff
@@ -686,18 +714,8 @@ where
     }
 }
 
-impl<T> DatabaseValue for T
-where
-    T: SerializableDatabaseValue + Decodable,
-{
-    fn from_bytes(data: &[u8], modules: &ModuleDecoderRegistry) -> Result<Self, DecodingError> {
-        T::consensus_decode(&mut std::io::Cursor::new(data), modules)
-            .map_err(|e| DecodingError::Other(e.0))
-    }
-}
-
 /// This is a helper macro that generates the implementations of
-/// [`DatabaseKeyPrefixConst`] necessary for reading/writing to the
+/// DatabaseRecord necessary for reading/writing to the
```

Comment:
nit: at some point add backticks around the type

Reply:
Whoops, missed this. Fixed now